apps/ledger-live-desktop/src/renderer/screens/nft/Gallery/TokensList/Item.jsx

```diff
@@ -60,9 +60,17 @@ type Props = {
   mode: "grid" | "list",
   withContextMenu?: boolean,
   onHideCollection?: () => void,
+  overloadOnItemClick?: (nftMetadata: NFTMetadata) => void,
```

Comment:
It's override not overload eheh

Also I would just have a `onItemClick?: (nftMetadata: NFTMetadata)` and rename the default one to `defaultOnItemClick`